redirects.json

```diff
@@ -1 +1 @@
-{"/documentation/":"http://devexpress.github.io/testcafe/documentation/getting-started/"}
\ No newline at end of file
+{"/documentation/test-api/selecting-page-elements/selectors.html":"http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/","/documentation/":"http://devexpress.github.io/testcafe/documentation/getting-started/"}
```

Comment:
I think we need to set redirect for all deleted pages, not only for `selectors.html`